_monitoring-plugins/alerting/api.md

```diff
@@ -1636,6 +1718,8 @@ POST _plugins/_alerting/destinations/email_groups
 Introduced 1.0
 {: .label .label-purple }
 
+When you update an email group, you can use the `seq_no` and `primary_term` to update an existing email group. If these numbers don’t match the existing email group or the email group doesn’t exist, alerting throws an error. OpenSearch increments the version number and the sequence number automatically (see the sample response).
```

Comment:
"When updating an email group, you can include `seq_no` and `primary_term` as part of your request. If these numbers don't match the existing monitor or the monitor doesn't exist, the alerting plugin throws an error. OpenSearch increments the version number and the sequence number automatically (see the sample response)."

Reply:
Thanks for the review, I fixed the description as you mentioned.